### PR TITLE
fix reloading of jobs & rotate logs

### PR DESCRIPTION
**Pull request by Fabio1988**
- fix reloading of jobs without restarting rotom
- allow to rotate log files stored to disk

Log format:
```
[2023-11-18T17:17:01.454Z] [INFO] [rotom] Rotom Device Controller Started
[2023-11-18T17:17:01.455Z] [INFO] [rotom] Device port 7070 Controller port 7071
[2023-11-18T17:17:01.455Z] [INFO] [rotom] Loaded 2 jobs
[2023-11-18T17:17:01.471Z] [INFO] [rotom] Webserver running on http://localhost:7072 listening on 127.0.0.1
```

### CODE REVIEW

**Comment by jfberry** [comment]
Does job reloading work with new files? I think perhaps it should track the folder including ‘add’

**Comment by Fabio1988** [comment]
We can also check for all file changes in that folder, even if a file is deleted we can include that change :)

**Comment by jfberry** [comment]
I think if you trace the folder you don’t include the * - a while since I played with chokidar

File: package.json [hunk 1]
```diff
@@ -43,6 +43,8 @@
     "tslib": "^2.4.0",
     "tslog": "^3.3.3",
```
Comment (na-ji) [inline package.json]: You can remove the dependency.

**Review by na-ji — APPROVED** [review]
LGTM

**Pull request merged**